### DISCUSSION TITLE
UI flicker issue on the autopay screen

### DISCUSSION THREAD

**Discussion by polkadirk** [OAK-Foundation/OAK-blockchain#500] [Automation Hub]
Have noticed a UI issue where the UI flickers - will attach a video and try to recreate and replicate.

https://github.com/OAK-Foundation/OAK-blockchain/assets/104081857/ad167543-d438-432b-8c06-1f629c6f0bbc

**Comment by v9n** [comment]
this is fixed now. no more flickering.
